Yoast/javascript
# Add words to the passive exceptions for English

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds the words 'annoyed', 'depressed', 'disappointed', and 'upset' to the list of English non-passives.
* [Yoast SEO Free] Does not mark the words 'annoyed', 'depressed', 'disappointed', and 'upset' as passives anymore, since most of the time they are closer to adjectives rather than passive verbs in meaning.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Set the site language to English
* Open a post.
* Paste sentences containing the words 'annoyed', 'depressed', 'disappointed', and 'upset', e.g. `I was annoyed. She is depressed. I am disappointed. We are upset.` Make sure they are not recognised as passives. The feedback should read `Passive voice: You're using enough active voice. That's great!`.
* Make sure that sentences that should be recognised as passives still are, for example `This glass has been broken. The fruit was eaten.` When you add these sentences to the text and click the highlight marker next to the passive voice assessment, these sentences should be highlighted.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
